units/SDL3_image.pas

```diff
@@ -0,0 +1,2220 @@
+unit SDL3_image;
+
+{
+  This file is part of:
+
+    SDL3 for Pascal
+    (https://github.com/PascalGameDevelopment/SDL3-for-Pascal)
+    SPDX-License-Identifier: Zlib
+}
+
+{*
+ * # CategorySDLImage
+ *
+ * Header file for SDL_image library
+ *
+ * A simple library to load images of various formats as SDL surfaces
+  }
+
+{$DEFINE SDL_IMAGE}
+
+{$I sdl.inc}
+
+interface
+
+  {$IFDEF WINDOWS}
+    uses
+      SDL3,
+      {$IFDEF FPC}
+      ctypes,
+      {$ENDIF}
+      Windows;
+  {$ENDIF}
+
+  {$IF DEFINED(UNIX) AND NOT DEFINED(ANDROID)}
+    uses
+      SDL3,
+      {$IFDEF FPC}
+      ctypes,
+      UnixType,
+      {$ENDIF}
+      {$IFDEF DARWIN}
+      CocoaAll;
+      {$ELSE}
+      X,
+      XLib;
+      {$ENDIF}
+  {$ENDIF}
+
+  {$IF DEFINED(UNIX) AND DEFINED(ANDROID) AND DEFINED(FPC)}
+    uses
+      SDL3,
+      ctypes,
+      UnixType;
+  {$ENDIF}
+
+const
+
+  {$IFDEF WINDOWS}
+    IMG_LibName = 'SDL3_image.dll';
+  {$ENDIF}
+
+  {$IFDEF UNIX}
+    {$IFDEF DARWIN}
+      IMG_LibName = 'libSDL3_image.dylib';
+      {$IFDEF FPC}
+        {$LINKLIB libSDL2}
```

Comment:
`libSDL3`

Reply:
My comment was about this bit here having `2` instead of `3`. But yeah, this should be `libSDL3_image`. Good catch.

Reply:
I was sure, I was missing something. :)